src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/IdentityRef.java

```diff
@@ -0,0 +1,114 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+public class IdentityRef implements Serializable {
+    private String displayName;
+    private String url;
+    @JsonProperty("_links")
+    private ReferenceLinks links;
+    private String id;
+    private String uniqueName;
+    private String directoryAlias;
+    private String profileUrl;
+    private String imageUrl;
+    @JsonProperty("isContainer")
+    private Boolean container;
```

Comment:
Possibly `boolean` rather than `Boolean` (same for other fields and method return types)?

Reply:
Fixed